azure-mgmt-trafficmanager/src/main/java/com/microsoft/azure/management/trafficmanager/TrafficManagerEndpoint.java

```diff
@@ -592,6 +696,66 @@
         }
 
         /**
+         * The stage of the traffic manager update definition allowing to specify the geographic region.
+         */
+        @Beta(Beta.SinceVersion.V1_2_0)
+        interface WithGeographicLocation {
+            /**
+             * Specifies the geographic location for the endpoint that will be used when the parent profile is configured
+             * with Geographic based routing method {@link TrafficRoutingMethod#GEOGRAPHIC}.
```

Comment:
the `@link` stuff does not translate well to C#, so it'd be better to avoid it. (I think just removing this whole section could work find here ---- just putting a dot after "routing method."